src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -29,8 +30,18 @@ public void actionPerformed(ActionEvent e) {
 				DrawerFeature.getDrawerController().clearPanel();
 				FiguresJoe.figureScript2(driverManager.getCurrentDriver());
 				break;
+			case "Factory Trinagle":
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.